fatal error: va/va_dec_av1.h: No such file or directory

Software that includes installed `va/va.h` can't compile: `va/va.h:4630:10: fatal error: va/va_dec_av1.h: No such file or directory`. Did you forget to add an install target for `va_dec_av1.h` for the automake toolchain?

This issue was introduced with #317

--- a/va/Makefile.am
+++ b/va/Makefile.am
@@ -42,6 +42,7 @@ libva_source_h = \
 	va_backend.h		\
 	va_backend_vpp.h	\
 	va_compat.h		\
+	va_dec_av1.h		\
 	va_dec_hevc.h		\
 	va_dec_jpeg.h		\
 	va_dec_vp8.h		\

